mobify/platform-scaffold
# Move SFCC connector

This PR removes the SFCC connector from the scaffold and updates references to it to use the SDK version. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-22
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/731

## Changes
- Remove SFCC connector
- Update paths for SFCC connector

## Todos

## How to test-drive this PR
- checkout the move-sfcc-connector branch for the SDK
- cd into the SDK folder
- run `npm link`
- run `npm run dev:build`
- cd into the scaffold
- run `npm link progressive-web-sdk`
- run `npm run dev`
- preview the SFCC site
- navigate throughout the site
- everything should work correctly
